deployment/hasura/migrations/AerieScheduler/2_reference_dataset_id_from_scheduling_request/up.sql

```diff
@@ -0,0 +1,4 @@
+alter table scheduling_request
+  add column dataset_id integer default null
+
+call migrations.mark_migration_applied('2');
```

Comment:
This migration needs to be marked as applied in the Scheduler's `applied_migrations.sql`.

Reply:
done

Reply:
Change was not pushed.

Reply:
now it should be